src/lib/CollisionPrevention/CollisionPreventionTest.cpp

```diff
@@ -130,6 +130,46 @@ TEST_F(CollisionPreventionTest, testBehaviorOnWithAnObstacle)
 	cp.modifySetpoint(modified_setpoint, max_speed, curr_pos, curr_vel);
 	orb_unadvertise(obstacle_distance_pub);
 
-	// THEN: it should be cut down a lot
-	EXPECT_GT(original_setpoint.norm() * 0.5f, modified_setpoint.norm()); //FIXME: this should actually be constrained to 0
+	// THEN: it should be cut down to zero
+	EXPECT_EQ(0.f, modified_setpoint.x());
```

Comment:
I think checking if the norm is 0 is probably easier, unless you were debugging this? Then you can fairly easily do
```suggestion
	EXPECT_EQ_FLOAT(0.f, modified_setpoint.norm()) << modified_setpoint.x() << "," << modified_setpoint.y();
```